### ISSUE TITLE
doubleTap doesn't trigger

### ISSUE THREAD

**Issue by justinboyd** [mattbryson/TouchSwipe-Jquery-Plugin#158]
Just want to say first that I love this plugin and thank you!

It looks like the doubleTap event isn't triggering the bound method. I have a super simple function and it still won't work on either desktop or mobile.

Fiddle: http://jsfiddle.net/justinboyd101/pqa2A/24/

**Comment by QuantumRevenger** [comment]
Great having seen this bug left for so long, it really does seem like another dev who just bails.

Like over 6+ months and a bug like this left untouched, I mean come on!

**Comment by mattbryson** [comment]
I agree, It's very rude of him not to keep this maintained for free at the expense of his full time job, other commercial projects that he works on, his family and other interests outside of maintaining this plugin.    It's an open source community project, so feel free to investigate it yourself and make a pull request.

**Comment by QuantumRevenger** [comment]
It kind of is given the fact the issue has been left unattended for so long and affects many people when you could easily set aside just an hour of your time over a 6+ month period; not like its been a few weeks.

Like you find the time to make a comment like this but for several months you don't even comment on the sheer volume of growing issues, just comes across like you can't be bothered anymore.

People rely on this plugin and when it doesn't work you cause them hassle, if you're not working on it anymore then make it clear on the readme; don't make people integrate a dead project.

**Comment by mattbryson** [comment]
Fair enough, Ill update the Read me with an indication on the time that I can put into the plugin, which at the moment Is about 2-3 days every 6 months.    An hour here and there wont cut it, takes a lot longer than that to get to the bottom of a lot of these browser  / OS based issues.  m

**Comment by tiesont** [comment]
@mattbryson Is it intentional that a tap handler must be defined for the doubleTap handler to work? If so, would you accept a pull-request to update the docs for http://labs.rampinteractive.co.uk/touchSwipe/docs/$.fn.swipe.html#event:doubleTap - the verbiage there seems to imply that it shouldn't be necessary, but an empty handler for tap was the only way I could get doubleTap to work.

If it's not intentional, and if you have a general idea of what is required to fix it, I could take a stab at updating the plugin, assuming you have time for a quick explanation.

**Comment by nofx77** [comment]
Found out that setting the treshold at 1 instead of 0 enable the tap and doubletap.
